cfitsio include not found: missing CFLAGS

I try to compile PI on an opensuse Tumbleweed system.  It fails compiling src/widgets/controls/controlwidget.cpp:
/usr/include/CCfits/CCfits.h:13:10: fatal error: fitsio.h: No such file or directory

Reason is the make call does not use the cfitsio/ccfits CFLAGS "-D_REENTRANT -I/usr/include/cfitsio"
Same happens for src/image_handlers/output_writers/imagefilewriter.cpp

--- a/cmake/find_dependencies.cmake
+++ b/cmake/find_dependencies.cmake
@@ -36,7 +36,7 @@ endif()
 
 # Cfitsio
 pkg_check_modules(CFITSIO REQUIRED cfitsio)
-include_directories(${CFITSIO_INCLUDEDIR})
+include_directories(${CFITSIO_INCLUDE_DIRS})
 message("CFITSIO: ${CFITSIO_LDFLAGS}")
 
 # LibUSB

--- a/src/widgets/configurationdialog.cpp
+++ b/src/widgets/configurationdialog.cpp
@@ -21,6 +21,7 @@
 #include "ui_configurationdialog.h"
 #include "commons/utils.h"
 #include <functional>
+#include <QButtonGroup>
 #include <QSpinBox>
 #include <QMessageBox>
 #include "Qt/qt_strings_helper.h"

--- a/src/widgets/recordingpanel.cpp
+++ b/src/widgets/recordingpanel.cpp
@@ -22,6 +22,7 @@
 #include "savefileconfiguration.h"
 #include "commons/messageslogger.h"
 #include "Qt/qt_functional.h"
+#include <QAction>
 #include <QDir>
 #include <QDialog>
 #include <QElapsedTimer>

